.github/workflows/build.yml

```diff
@@ -164,12 +164,7 @@ jobs:
         echo "::add-path::${{github.workspace}}\build\openvdb\Release"
     - name: update_vcpkg
       shell: bash
-      run: |
-        # temporarily update vcpkg to fix tbb port
-        cd C:\vcpkg
-        git fetch
-        git checkout 97302ca1729eac9917c8a8977667601453795c90
-        bootstrap-vcpkg.bat
+      run: cd C:\vcpkg && git pull
```

Comment:
always update available ports (this is actually needed for the blosc external deps where they've recently changed the zstd library name)

Reply:
Is this always needed or is this needed just until that change makes it way into the default vcpkg? I'd prefer this doesn't run from a different commit each time so that it's easier to reproduce and debug issues.

Reply:
It was necessary but I would anticipate I can now remove it - as you say, updating on each commit isn't ideal